Assignment/DS3/Match.java

```diff
@@ -0,0 +1,78 @@
+
+import java.util.ArrayList;
```

Comment:
This should be implemented by queue. Assignment should be implemented with the topic i.e covered in the class.